internal/uri/uri_test.go

```diff
@@ -178,3 +178,36 @@ func TestMustParseURI(t *testing.T) {
 		})
 	}
 }
+
+func TestIsWSLURI(t *testing.T) {
+	type args struct {
+		uri string
+	}
+	tests := []struct {
+		name string
+		args args
```

Comment:
```suggestion
		args string
```
Is the extra struct here necessary, since it's just one argument and a string?